app/views/handlers/content.py

```diff
@@ -33,7 +33,8 @@ def get_context(self):
                 self._schema.get_individual_response_list(),
                 self._current_location.list_item_id,
                 self._questionnaire_store,
-            ),
+            )
+            and show_individual_response_link(self._current_location, self._schema),
```

Comment:
You can either check `show_individual_response_link` in templates or set `individual_response_url` to None when `show_individual_response_link` is False. The first way is more explicit.
Any preference @iwootten?

Reply:
I think the best thing would be for the value of show_individual_response_link (as `show_individual_response`) is passed to the interstitial/hub templates and that is used to determine visibility. We should update my pr changes following it's merge so it factors into the value of show_individual_response_link

Reply:
For what it's worth, I think my PR and this should have been done as a single card. They both require thinking about when we decide to show the link.

Reply:
So in your recent change, for the hub we passed the URL to the templates only when we know we can show it. Are you suggesting that should be changed, and we should pass, both the URL and show_ir flag?

Reply:
I don't mind either way - I think passing the flag through makes sense and I don't actually think we should be setting the url attribute at all if we don't have one. I backtracked on this approach on my own pr though as I think there's some subtleties to how the url is generated that may cause issues.

Reply:
Making `individual_response_url` visible on `show_individual_response` flag set to True is breaking all sorts of tests, seems like `individual_response_url` is needed in many other places that use handlers __init__ methods, it's not only used by interstitial pages. Or should we change that `show_individual_response` method so it evaluates something else then first block of individual section?

Reply:
I think that's likely the reason why I gave up on that approach - I say just go with the appraoch @MebinAbraham has suggested then

Reply:
I tried your change @MebinAbraham  and I get `individual_response_url` evaluated to None when I try to access IR link from remove person screen, so what should be `show_individual_response` method check, other than first block of individual section? This is the only test it currently breaks on.
<img width="670" alt="Screenshot 2020-09-29 at 13 14 38" src="https://user-images.githubusercontent.com/53475968/94557210-2c621300-0256-11eb-8911-a732359a049f.png">

Reply:
Done!